tests/testthat/_snaps/author.md

```diff
@@ -3,33 +3,33 @@
     Code
       challenge_legacy_author_fields()
     Message
-      x Found legacy Author and/or Maintainer field in DESCRIPTION.
-        usethis only supports modification of the Authors@R field.
+      x Found legacy 'Author' and/or 'Maintainer' field in DESCRIPTION.
+        usethis only supports modification of the 'Authors@R' field.
       i We recommend one of these paths forward:
-        * Delete these fields and rebuild with `use_author()`.
-        * Convert to Authors@R with `desc::desc_coerce_authors_at_r()`,
-          then delete the legacy fields.
+      [ ] Delete the legacy fields and rebuild with `use_author()`; or
+      [ ] Convert to 'Authors@R' with `desc::desc_coerce_authors_at_r()`, then delete
+        the legacy fields.
     Condition
-      Error:
-      ! User input required, but session is not interactive.
-      Query: Do you want to cancel this operation and sort that out first?
+      Error in `ui_yep()`:
```

Comment:
Now that usethis uses `rlang::abort()` indirectly, it becomes obvious that I also haven't yet threaded the caller through everywhere. That is a separate job.